src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -757,8 +757,7 @@ public void afterAnalysis(
   // Separating the conditions for readability.
   private boolean shouldParseNoCacheOutputs() {
     if (remoteOptions == null
-        || remoteOptions.remoteBuildEventUploadMode != RemoteBuildEventUploadMode.ALL
-        || !remoteOptions.incompatibleRemoteBuildEventUploadRespectNoCache) {
```

Comment:
@coeuvre The way this was written, using one flag or the other, meant that this always evaluated to `true` (so always returned `false`). The tests are breaking with this change, and to get the hold behavior, I would instead change this to `remoteOptions == null || true`, or just have the function always return `false`.

Not sure what to do for this.

Reply:
always return `false` SGTM.